proto/dymensionxyz/dymension/rollapp/rollapp.proto

```diff
@@ -53,6 +53,8 @@ message Rollapp {
   // register and serve for this rollapp. if left empty, no sequencer is allowed
   // to register. if set to "*" any sequencer can register.
   string initial_sequencer = 13;
+  // how much DYM a sequencer needs (at least) to be proposer
```

Comment:
is this field set in base denom (i.e adym) or DYM?

Reply:
its dym
please dont make me change it

Reply:
@danwt why make it different than other params? they all adym.
now u also need this on the caller side:
```
func (k Keeper) minBond(ctx sdk.Context, rollapp string) sdk.Coin {
	nDym := k.rollappKeeper.MinBond(ctx, rollapp)
	ret := commontypes.DYMCoin
	ret.Amount = ret.Amount.Mul(nDym)
	return ret
}
```
even better, it should be `sdk.Coin`, as in other relevant params (kick threshold e.g)

Reply:
reworked